src/cpp/session/SessionMain.cpp

```diff
@@ -409,6 +410,9 @@ Error rInit(const rstudio::r::session::RInitInfo& rInitInfo)
    
       // client event service
       (startClientEventService)
+      
+      // rpc methods
+      (rpc::initialize)
```

Comment:
I feel like we might want to call this earlier on in the startup process, but wasn't quite sure where the best place to fit this might be.

Reply:
I think this is safe, but if you're concerned about it you could wrap the static pointer in an accessor function which returns the instance and creates it on demand if necessary.

Reply:
I thought about doing that as well, but wanted to be a bit more deterministic in when initialization occurred. This should be sufficient then -- thanks for the review!